server/pages/index.js

```diff
@@ -119,7 +140,7 @@ function getCurrentCommitShortChecksum() {
 	}
 }
 
-function getDefaultContext( request ) {
+function getDefaultContext( request, callback ) {
```

Comment:
Hmmm. We could use either.

The main reason I used Promises for reading the files was so that I could easily start the process of reading both files at once, and then use `Promise.all` to wait for the contents of both files to be available. Trying to do that with callbacks is tricky, and the alternative is to wait for one file to be finished loading before starting to read the second one.

I didn't have a compelling reason to have `getDefaultContext` return a Promise, so I just used a callback. But it certainly could return a Promise instead.